client/src/features/gPromises/GPromise.tsx

```diff
@@ -131,13 +149,23 @@ export default function GPromiseContainer({ gPromise }: GPromiseProps) {
   const passage = formatPassage(text);
   return (
     <Article>
-      <Header>
-        <Title>{reference}</Title>
-        <Subtitle>{bibleName}</Subtitle>
-      </Header>
-      <BlockquoteWrapper>
-        <Blockquote>{passage}</Blockquote>
-      </BlockquoteWrapper>
+      <SwitchTransition>
+        <CSSTransition
+          key={text}
```

Comment:
It will also apply the transition if we switch languages. 🤔 Not sure about this.